### ISSUE TITLE
Snap store not following whitesur theme while other snaps do

### ISSUE THREAD

**Issue by FlyinPancake** [vinceliuice/WhiteSur-gtk-theme#340]
OS and stuff (output of `snap version`):
```
snap    2.51-1.fc34
snapd   2.51-1.fc34
series  16
fedora  34
kernel  5.12.13-300.fc34.x86_64
```
Screenshot 1:
![image](https://user-images.githubusercontent.com/36113055/124099180-8ac65700-da5d-11eb-81ad-7d12b713a934.png)
Here you can see the snaps I installed (`snap-store` and `code-insiders`) and both of them are open on the right hand side. code insiders follows the theme but snap store doesn't. (Is it because `snap-store` is based on `core18` instead of `core20`?)

If more explanation is needed just reply I'll try to help out any way I can.

**Comment by FlyinPancake** [comment]
Icon theme isn't working for snaps as well, but for system apps and flatpaks it works

**Comment by rivanfebrian123** [comment]
> Here you can see the snaps I installed (snap-store and code-insiders) and both of them are open on the right hand side. code insiders follows the theme but snap store doesn't. (Is it because snap-store is based on core18 instead of core20?)

Actually, there's nothing to do with that. VSCode use Electron instead of GTK. Anyway, did you run `./tweaks.sh -s` already?

**Comment by FlyinPancake** [comment]
Yes I did but snap store seems not to respect the gtk theme at all.
Heres my 1234 th time doing it: 

https://user-images.githubusercontent.com/36113055/124278298-63978480-db46-11eb-82d2-6699c2315c52.mp4

**Comment by rivanfebrian123** [comment]
Usually, some apps still run in background even when you "close" it. Can you please logout and open it again?

**Comment by FlyinPancake** [comment]
Still nothing changed :/

**Comment by popykunju** [comment]
I even had this problem,there seems to be a solution like link theme inside the snap themes folder in root
